src/pspm_cfg/pspm_cfg_run_find_sounds.m

```diff
@@ -28,9 +26,7 @@
       options = pspm_update_struct(options, d.create_corrected_chan.yes, 'channel_action');
       options.channel_output = 'corrected';
     end
-    if isfield(d.marker_chan, 'marker_nr')
```

Comment:
This seems to be a bug fix. I tested the GUI by specifying some values but I cannot see `d.marker_chan` defined. I think the current change can be delivered correctly.


```
d = 
  struct with fields:
              diag_output: [1×1 struct]
    create_corrected_chan: [1×1 struct]
                     chan: [1×1 struct]
                max_delay: 3
                min_delay: 0
                 n_sounds: 0
```
```
d.chan
ans = 
  struct with fields:
    chan_nr: 1
```